x-pack/test/api_integration/apis/security_solution/timeline_migrations.ts

```diff
@@ -44,14 +46,14 @@ export default function ({ getService }: FtrProviderContext) {
         await esArchiver.load(
           'x-pack/test/functional/es_archives/security_solution/timelines/7.15.0_space'
         );
-      });
-
-      after(async () => {
-        await esArchiver.unload(
```

Comment:
What will happen if we unload the es archive? Not unloading it might affect other tests because we'd be preserving the data right?

Reply:
Yeah almost always better to unload it or never load it in the first place lol